wordpress-mobile/WordPress-iOS
# Replace "More" button behavior to show options in an Action Sheet

Ref #11646

This is the first task of the Post list revamp - it aims to change the behavior of the "More" button.

* Each post cell now presents a "More" action
* When tapped, shows an Action Sheet with available options

The available options are as following:

* Published post: "Stats", "Move to Draft", "Move to Trash"
* Drafted post: "Stats", "Move to Trash"
* Scheduled post: "Move to Draft", "Move to Trash"
* Trashed post: "Move to Draft", "Delete Permanently"

<p align="center">
  <img src="https://user-images.githubusercontent.com/7040243/57422162-46283380-71e5-11e9-8741-c2d6b7b6057c.png" alt="" width="400" />
</p>

To test:

* Go to Blog Posts
* Checks that each cell displays a "More" action
* Tap it and verify that all available options work as expected

Additional info:

The `PostCardActionBar` wasn't removed yet. This will be done with the update of the card design which will make this entity not needed anymore.

Update release notes:
